packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+import { range } from 'lodash';
```

Comment:
I'm a bit surprised that this isn't causing any issues since `lodash` isn't defined as a dependency in the web-components manifest:

![image](https://user-images.githubusercontent.com/2008881/144670060-66adfcbe-a2f9-4d2c-af24-574cdf010c38.png)


Maybe there's some magic going on to make it available through the monorepo since it's used in `react-components`. Anyway, I think we can [use some native JS](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_range) for this. And I know that the existing React `SegmentedProgressBar` is using lodash, but I don't think we need to carry that over unless absolutely necessary.

We already have polyfills for `Array.from` in IE11, so that shouldn't be an issue.

Reply:
Updated